projects/spectator/schematics/package.json

```diff
@@ -3,9 +3,8 @@
   "version": "0.0.0",
   "schematics": "./src/collection.json",
   "peerDependencies": {
-    "@angular-devkit/core": "^7.3.1",
-    "@angular-devkit/schematics": "^7.3.1",
-    "schematics-utilities": "^1.1.1",
-    "typescript": "~3.1.1"
+    "@angular-devkit/core": ">= 8.0.0",
```

Comment:
Why this change?

Reply:
To be consist with https://github.com/ngneat/spectator/blob/master/projects/spectator/package.json